semopenalex-app/data/templates/Template%3Ahttps%3A%2F%2Fsemopenalex.org%2Fclass%2FInstitution.html

```diff
@@ -1,100 +1,357 @@
+  }
+  .publications .card__title {
+    text-transform: uppercase;
+  }
+  .tab-pane > div {
+    padding-bottom: 2rem;
   }
-
-  .concept {
-  background-color: #ddd;
-  border: none;
-  color: black;
-  padding: 10px 20px;
-  text-align: center;
-  text-decoration: none;
-  display: inline-block;
-  margin: 4px 2px;
-  cursor: pointer;
-  border-radius: 16px;
-}
-
-
 </style>
 
-<ol class="page-breadcrumb">
-    <li>
-      <semantic-link title="Start" iri='{{rdf-resolvePrefix "semopenalex:Start"}}'>Start</semantic-link>
-    </li>
-    <li class="active">Institution</li>
-</ol>
-<div class="page">
-  <div class='page__body'>
-
-    <div style="margin: 8px;">
-      <div class="card-group">
-        <div class="card" style='border: none;'>
-          <mp-resource-thumbnail iri='{{page-resource}}' style="height: 100px; width: 100px;  object-fit: contain;">
-          </mp-resource-thumbnail>
-                             
-          <mp-label iri="{{page-resource}}"></mp-label>
-                            
-          <div class="card-body">
-
-            <div style='padding:16px'>
+<template-fragment id='Cards'>
+  <div class="cards">
+    <div class="card publication-chart">
+      <div>
+        <div>
+          <div class="publications__label">Publications</div>
+          <semantic-query query='
+            SELECT ?count WHERE { <{{page-resource}}> <https://semopenalex.org/property/worksCount> ?count}' template='{{> tmp}}'>
+              <template id="tmp">
+                {{#each bindings}}
+                  <div class="publications__value">{{count.value}}</div>
+                {{/each}}
+              </template>
+          </semantic-query>
+        </div>
+        <div>
+          <div class="citations__label">Citations</div>
+          <semantic-query query='
+            SELECT ?cited WHERE {
+	            <{{page-resource}}> <https://semopenalex.org/property/citedByCount> ?cited
+            }' template='{{> tmp}}'>
+              <template id="tmp">
+                {{#each bindings}}
+                  <div class="citations__value">{{cited.value}}</div>
+                {{/each}}
+              </template>
+          </semantic-query>
+        </div>
+      </div>
+      <semantic-chart type="bar" query="
+        SELECT ?year ?work WHERE {
+          <{{page-resource}}> <https://semopenalex.org/property/countsByYear> ?count .
+          ?count <https://semopenalex.org/property/year> ?year .
+          ?count <https://semopenalex.org/property/worksCount> ?work .
+        } ORDER BY ASC(?year)
+      " sets='[{"dataSetName": "Publications", "category": "year", "x": "year", "y": "work"}]'
+        styles='[{
+          "provider": "highcharts",
+          "style": {
+            "series": [
+              {"type": "column", "showInLegend": false}
+            ]
+          }
+        }]'
+        style-highcharts='{
+          "chart": {
+            "margin": [20, 0, 20, 40]
+          },
+          "legend": "hide"
```

Comment:
i think we need a bit of clear for visualization by saying that it only shows the last 10 years of publications. 
let's use with `subtitle` option:
```
 "subtitle": {"text": "The last 10 years of publications."}
```